apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerConversionOptionsTests.cs

```diff
@@ -67,35 +67,41 @@ public void WithSpannerToClrMappings()
         public void WithClrToSpannerMappings()
         {
             var options = SpannerConversionOptions.Default;
-            options = options.WithClrToSpannerMappings("DecimalToNumeric,DateTimeToDate");
+            options = options.WithClrToSpannerMappings("SingleToFloat32,DecimalToNumeric,DateTimeToDate");
```

Comment:
(The line isn't relevant here.)

Is there already a test for the defaults somewhere? It feels like a test that just uses SpannerConversionOptions.Default and asserts all the defaults would be useful. It's probably covered by the other test classes, but it feels like it would be useful documentation.

Reply:
I've added a test for defaults, nice catch, there was none explicitly.